### PR TITLE
Added 2 new settings to PCA Configuration

### PR DESCRIPTION
**Pull request by shoatman**
- concurrent_authorization_requests (boolean, default: false) - controls whether concurrent authorization requests are allowed
- authorization_in_current_task  (boolean, default:false)-  controls whether authorization occurs in the same task as the activity that started interactive request or occurs in a new task
- concurrent_authorization_requests setting is used in initializeConfigurationInternal to configure the command dispatcher to allow a new max threads in the interactive thread pool

### CODE REVIEW

File: msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfigurationFactory.java [hunk 1]
```diff
@@ -50,6 +52,7 @@
 
 public class PublicClientApplicationConfigurationFactory {
     private static final String TAG = PublicClientApplicationConfigurationFactory.class.getSimpleName();
+    private static final int DEFAULT_MAX_THREAD_POOL_INTERACTIVE_CONCURRENT = 5;
```
Comment (gitaumoses4) [inline msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfigurationFactory.java:55]: Just curious is there any specific reason for choosing 5 here?

Comment (shoatman) [reply msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfigurationFactory.java:55]: Nope.  Open to suggestion, but didn't seem reasonable for it to be unlimited.  We used 5 for the silent pool in command dispatcher, so i made it match.

**Review by gitaumoses4 — APPROVED** [review]
Looks good. No more comments.

File: msal/build.gradle [hunk 2]
```diff
@@ -298,6 +298,7 @@ dependencies {
         transitive = false
     }
 
+
 }
```
Comment (iamgusain) [inline msal/build.gradle:302]: nit: remove

**Review by iamgusain — APPROVED** [review]
:shipit:

**Pull request closed without merging**